definitions/scenarios/restore.rb

```diff
@@ -18,6 +18,7 @@ def compose
       add_steps_with_context(Checks::Restore::ValidateBackup,
                              Procedures::Restore::Confirmation,
                              Checks::Restore::ValidateHostname,
+                             Checks::Restore::ValidateInterfaces,
```

Comment:
How does the error reporting work? If both the hostname and the interfaces are incorrect, does report both error or does it quit after the first check fails?

I'm asking because I want to have a check later on for the Smart Proxy Realm feature. Ideally we would report all errors at the same time so the user doesn't get in a loop of fix problem, next restore, fix next problem, etc. That means we may need to have a check `ValidateAnswersMatchTargetSystem` or something like that.

Reply:
My understanding that today it quits after the first error, but I might be wrong.

Reply:
@evgeni yes it quits after first error as default run_strategy is fail_fast. I think the restore scenario needs implementation like upgrade scenario. So in that case the all the pre-restore checks will be in one seperate scenario and that will be having fail_slow run strategy.

Reply:
Yepp, sounds like a good follow-up!